frontend/src/app/main/ui/context.cljs

```diff
@@ -29,3 +29,4 @@
 
 (def workspace-read-only? (mf/create-context nil))
 (def is-component?        (mf/create-context false))
+(def sidebar (mf/create-context nil))
```

Comment:
Although in theory I _guess_ we could rely on `show-recent`, if later on we decide to show recent fonts somewhere else, it could break the selector, since atm positioning heavily depends on this being on the right side bar.

It's one of these things that we could refactor later on.

If we are sure the only place to show recent fonts is that specific font selector, and it will be kept like that, we could drop this context.